internal/commands/chat-sast.go

```diff
@@ -44,8 +45,11 @@ func ChatSastSubCommand(chatWrapper wrappers.ChatWrapper) *cobra.Command {
 	return chatSastCmd
 }
 
-func runChatSast(chatWrapper wrappers.ChatWrapper) func(cmd *cobra.Command, args []string) error {
+func runChatSast(chatWrapper wrappers.ChatWrapper, tenantWrapper wrappers.TenantConfigurationWrapper) func(cmd *cobra.Command, args []string) error {
 	return func(cmd *cobra.Command, args []string) error {
+		if !aiGuidedRemediationEnabled(tenantWrapper) {
```

Comment:
@tiagobcx - when I validate here - we will get the validation: 
_required flag(s) "chat-apikey", "sast-result-id", "scan-results-file", "source-dir" not set_
before the aiGuidedRemediationEnabled  setting validation